docs/releasing/make-docker-images.sh

```
@@ -40,7 +40,19 @@ while IFS= read -d $'\0' -r dir; do
     (
         cd "${builddir}"
         log "Building (and pushing) image on Google Cloud Build: ${image}"
-        gcloud builds submit --project=${PROJECT_ID} --tag=${image}
+        log "Submitting Cloud Build job..."
+        build_id=$(gcloud builds submit --project=${PROJECT_ID} --tag=${image} --async --format="value(id)")
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)